<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPainter</class>
 <widget class="QMainWindow" name="MyPainter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyPainter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar_figures">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_rectangle"/>
   <addaction name="separator"/>
   <addaction name="action_circle"/>
   <addaction name="separator"/>
   <addaction name="action_triangle"/>
   <addaction name="separator"/>
   <addaction name="action_line"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="tool_bar_propereties">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="action_rectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/rect.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="action_circle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/circle.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="action_triangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/triang.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="action_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/line.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
